ee_tests/README.md

```diff
@@ -50,133 +50,4 @@ implement the test.
 npm start local
 ```
 
-## The rest of the documentation should be reviewed
-
-
-
-##### Debugging #####
-
-You can use chrome devtools to pause and debug typescript tests by inserting
-the `debugger` statement in the spec and running the test with `NODE_DEBUG`
-environment flag set. E.g.
-
-```
-NODE_DEBUG=true npm start
-```
-
-##### Turn Debug Statements on #####
```

Comment:
Why do you remove the DEBUG usage docs?
It is still present in the tests and it's a useful feature.
Can you keep it in the README, please (if there's no particular reason to remove it)?